.travis.yml

```diff
@@ -2,9 +2,6 @@ dist: trusty
 sudo: required
 language: node_js
 
-services:
-  - docker
-
```

Comment:
This isn't necessary? I thought it was necessary because we do `docker pull` and such.

Reply:
I thought so too!

Check out one of the jobs from the latest develop build: https://travis-ci.org/trufflesuite/truffle/jobs/507968192#L415

Reply:
Ah, looks like it's because our builds use `trusty` w/ `sudo: required`.